blog/routes/web.php

```diff
@@ -13,8 +13,26 @@
 
 Route::get('/', function () {
     return view('welcome');
-});
-Route::get('/user/{id?}', function($id = 'd'){
-    return "welcome ".$id;
+})->name('default')->middleware('logging');
+
+Route::get('/welcome/{age}', function($age){
+    return "welcome ".$age;
+})->middleware('logging')->middleware('ageRestriction');
+
+Route::get('/user/{id}', 'HomeController@getUser')->name('getUser')->where('id', '[0-9]+')->middleware('logging');
```

Comment:
sobzirom da imas rute koje koriste isti middleware , hajde da uzmemo i da grupisemo to. Evo dokumentacije https://laravel.com/docs/master/middleware#middleware-groups , ako je nesto nejasno cimaj

Reply:
ovo ispravi